web/src/components/utils/SelectContext/SelectCheckbox.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * Panther is a Cloud-Native SIEM for the Modern Security Team.
+ * Copyright (C) 2020 Panther Labs Inc
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import { Checkbox } from 'pouncejs';
+import { useSelect } from './SelectContext';
+
+interface SelectCheckboxProps {
+  id: string;
```

Comment:
Wee should change this since it clashes with the native HTML  `id` property. Thus, if a developer wants to add an HTML `id` to this `Checkbox`, he/she now can't. By renaming it to `selectionId` we avoid clashes like that

```suggestion
  selectionId: string;
```